www/kuza13/sp2/src/index.js

```diff
@@ -0,0 +1,122 @@
+const taskInput = document.querySelector('#taskInput');
+const form = document.querySelector('#form');
+const tasksList = document.querySelector('#tasksList');
+
+let tasks = [];
+
+// Button listeners
+
+form.addEventListener('submit', addTask);
+tasksList.addEventListener('click', removeTask);
+tasksList.addEventListener('click', completeTask);
+
+if (localStorage.getItem('tasks')) {
+	tasks = JSON.parse(localStorage.getItem('tasks'));
+	tasks.forEach(task => renderTask(task));
+}
+
+checkEmptyList(tasksList);
+
+// Functions
+
+async function addTask(event) {
+	event.preventDefault();
+
+	const taskText = taskInput.value;
+
+	const task = {
+		id: Date.now(),
+		text: taskText,
+		done: false,

```

Comment:
pridejte vic atributu, napr. planovany datum, priorita, ... to same do formulare pro vytvareni tasku